wcomponents-examples/src/main/java/com/github/bordertech/wcomponents/examples/picker/SafetyContainer.java

```diff
@@ -152,7 +152,7 @@ protected void paintComponent(final RenderContext renderContext) {
 			} else {
 				try {
 					for (int i = 0; i < shim.getChildCount(); i++) {
-						shim.getChildAt(i).paint(renderContext);
+    						shim.getChildAt(i).paint(renderContext);

```

Comment:
Style nit: please use consistent indentation characters (i.e. tabs)
@jonathanaustin looks like your checkstyle rules are missing this scenario.